src/examples/fortran_hello_world.f90

```diff
@@ -0,0 +1,46 @@
+module LibTrixi
```

Comment:
I think the module that describes the C interface should go into its own file, such that it is usable outside the example.